adapters/rise/risetest/exemplary/banner-and-video-site.json

```diff
@@ -18,7 +18,7 @@
         },
         "ext": {
           "bidder": {
-            "org": "72721",
+            "publisher_id": "72721",
```

Comment:
Requesting you to please add at least one test case where org is not empty. Your code assumes that if both org and publisherID is provided, publisherID takes the precedence. You might wanna consider reversing your if condition; since org should take the precedence but it is for you to decide.
I would like to make a request to please [update the docs](https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/rise.md) with above information(which one should take the precedence) and link the PR here.

Reply:
The code takes `org` first if present. I have added a test case to show that.

For the docs, it is okay for us that they stay as they are. As you can see, the docs already mention the `org` field we are trying to transition to here.

Reply:
@zkosanovic thanks for being patient here. 
If you add both org and publisherID, publisherId takes the precedence. You can also add a test case to validate this. For example - 
```
"ext": {
          "bidder": {
            "publisher_id": "1",
            "org": "2",
            "path": "mvo",
            "zone": "1r"
          }
          ```
In above case, both org and publisher_id is provided but value of publisher_id will take precedence. 

For the docs, I am ok if you would not like to mention about the publisher_id.

Reply:
Cool! Thanks for that test case. g2g.